File: 0771) Jewels and Stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        int ans = 0;
        unordered_set<char> j(jewels.begin(), jewels.end());
        for(char i : stones)
        {
            if(j.count(i))
            {
                ans++;
            }
        }
        return ans;
    }
};

File: 0771) Jewels and Stones.java
class Solution {
    public int numJewelsInStones(String jewels, String stones) {
        int ans = 0;
        HashSet<Character> set= new HashSet<>();
        for(char ch : jewels.toCharArray())
        {
            set.add(ch);
        }
        for(char ch : stones.toCharArray())
        {
            if(set.contains(ch))
            {
                ans++;
            }
        }
        return ans;
    }
}
